src/covid_model_seiir_pipeline/paths.py

```diff
@@ -272,6 +273,10 @@ def get_location_dir(self, location_id: int):
             folder = matches[0]
         return self.root_dir / folder
 
+    def get_modelled_locations(self) -> List[int]:
```

Comment:
Would be better if we had Aaron write out a `metadata.yml` file in the future rather than scraping the folder names

Reply:
I've been thinking about locations a bit.  Low priority, but good note.